SystemDocumentation/ProjectReport.txt

```diff
@@ -0,0 +1,63 @@
+Project Report.
+
+First we will go over what we have delivered, we will go over each priority and tell what we have delivered.
+
+High Priority:
+We have output that is readable, but we weren't able to get it into csv format for inputting into Excel
+As a manager you are able to generate weekly sales and profit reports.
+As a manager you can input sale prices from the manager page.
+As a manager you are able to override products and set them to what the manager wants.
+
+Medium Priority:
+We were able to get (semi) automatic purchase orders, whenever a product has lower than 5 items in the inventory it will ask if you want to order more
+Every transaction is saved in the database, and can be used by the manager
+You can order more products in case of spoiled/stolen products from the manger page
+We are able to search the inventory from the manager and the cashier page, but we don't have a method of sorting it.
+
+Low Priority:
+We weren't able to tell when products expired/spoiled
+We do have user modes, the a person that is a cashier can access the POS cashier page, but they can't access the manager page.
+A set of manager credentials can access the manager page and the cashier page so they can do both. You can tell who's credentials are
+mangers within the database.
+
+We will now give a brief overview of the product.
+
+The product runs on https://cryptic-crag-17136.herokuapp.com
+
+Which will ask you to log in, you can log into this with your assigned
+username and password
+
```

Comment:
Probably worth noting, when you log in using manager creds, you will be redirected to the manager page